From 9a75327ddce479d76ab0c45d05e62b180f96f28e Mon Sep 17 00:00:00 2001
From: Jacob Champion <jacob.champion@ni.com>
Date: Tue, 2 Sep 2014 11:27:25 -0500
Subject: [PATCH 19/22] Don't use wincrypt.h on PharLap

The no-ssl version of libcurl uses Windows CryptoAPI for its MD5
implementation, but those entry points don't exist on PharLap. Use the
fallback instead.

Upstream-Status: Inappropriate [Pharlap specific]
---
 lib/md5.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/md5.c b/lib/md5.c
index f818d32..d2d960b 100644
--- a/lib/md5.c
+++ b/lib/md5.c
@@ -124,7 +124,7 @@ static void MD5_Final(unsigned char digest[16], MD5_CTX *ctx)
   CC_MD5_Final(digest, ctx);
 }
 
-#elif defined(_WIN32) && !defined(CURL_WINDOWS_APP)
+#elif defined(_WIN32) && !defined(CURL_WINDOWS_APP) && !defined(PHARLAP)
 
 #include <wincrypt.h>
 #include "curl_memory.h"
-- 
2.10.2

